.devops/deploy-pipelines.yml

```diff
@@ -311,4 +342,230 @@ stages:
                     NAMESPACE: ecommerce
                     APP_NAME: $(K8S_IMAGE_REPOSITORY_NAME)
                     VALUE_FILE: "helm/values-uat.yaml"
-                    GREEN_VERSION: $(green_version)
\ No newline at end of file
+                    GREEN_VERSION: $(green_version)
+# --- END Deploy UAT --- # 
+
+# --- START Deploy PROD --- # 
+  - stage: "Prod_WaitForApproval"
+    displayName: 'PROD approval deployment'
+    dependsOn: [Deploy_UAT_Green,Release]
```

Comment:
Should here add a condition that both task executed successfully? Seems that they are in or by default where just one has to be completed successfully for the other to be executed (see [this run](https://dev.azure.com/pagopaspa/pagoPA-projects/_build/results?buildId=100604&view=results) where manual prod task was canceled for a script error)